contentcuration/contentcuration/frontend/channelEdit/components/edit/__tests__/learningActivityOptions.spec.js

```diff
@@ -0,0 +1,22 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import { shallowMount } from '@vue/test-utils';
+import LearningActivityOptions from '../LearningActivityOptions.vue';
+import { LearningActivities } from 'shared/constants';
+
+Vue.use(Vuetify);
+
+describe('LearningActivityOptions', () => {
+  it('smoke test', () => {
+    const wrapper = shallowMount(LearningActivityOptions);
+    expect(wrapper.isVueInstance()).toBe(true);
+  });
+
+  it('number of items in the dropdown should be equal to number of items available in ', () => {
+    const wrapper = shallowMount(LearningActivityOptions);
+    const numberOfDropdownItems = Object.keys(LearningActivities).length;
+    const dropdownItems = wrapper.attributes()['items'].split(',').length;
+
+    expect(dropdownItems).toBe(numberOfDropdownItems);
+  });
```

Comment:
Maybe a quick smoke test of the v-model behaviour of the component? That's the main interface here that we want to be reliable.

Reply:
Can I mount a fake parent component and check to see that the value is getting passed?

```
  it('updates a parent with its v-model', () => {
    const parent = mount({
      data: { activity: [] },
      template: '<div> <LearningActivityOptions v-model="activity"></LearningActivityOptions> </div>',
      components: { 'LearningActivityOptions': LearningActivityOptions }
    });

    const dropdown = parent.find('input');
    dropdown.element.value = 'Explore';
    dropdown.trigger('input');

    expect(parent.vm.activity).toBe('Explore');
  });
```

Reply:
Yep that looks like there's right shape but will need some other tweaking to set the input.